Dockerfile

```diff
@@ -3,5 +3,5 @@ WORKDIR /app
 COPY package.json /app
 RUN npm install
 copy . /app
-CMD node myapp.js
+CMD [ "node", "app.js" ]
 EXPOSE 3009
```

Comment:
I originally didn't see the port specified in the example app, however I realized I was looking at the "the-example-app.nodejs" repo, rather than the "Nodejssample" repo.  I did see the listen port specified in that repo and it was specified as 3009, so you can ignore my earlier comments about confusion regarding the port.